lib/nolibc/musl-imported/src/string/strstr.c

```diff
@@ -0,0 +1,156 @@
+/* taken from musl */
+
+#include <string.h>
+#include <stdint.h>
+
+static char *twobyte_strstr(const unsigned char *h, const unsigned char *n)
```

Comment:
Can we refactor those function to make something more generic?

Reply:
I generally agree with your points here. And although yes, musl is under MIT license, allowing a great amound of freedom, including publishing modified versions, I would still keep it as is :thinking: